source/clear-linux/guides/maintenance/swupd-mixer-integration.rst

```diff
@@ -0,0 +1,155 @@
+.. _swupd-mixer-integration:
+
+Mixer tool enhancement
+######################
+
+Mixing means composing an operating system for a specific use case. The
+default build of |CLOSIA| provides options to install bundles for various
+server capabilities. If you want to supplement the default OS with
+functionality from other OS distributions, you can manually compose an OS
+using :ref:`the mixer tool <mixer>`.
```

Comment:
Open: if existing topic mixer.rst will be eliminated, this pointer must be deleted.

Reply:
The topic has not been eliminated, on the contrary it was updated and the cross-reference should stay in place.